collab-database/src/rows/row.rs

```diff
@@ -305,7 +305,7 @@ impl Row {
       height: DEFAULT_ROW_HEIGHT,
       visibility: true,
       created_at: 0,
-      modified_at: 0,
```

Comment:
what is the purpose rename modified_at -> last_modified? it seems the same. If some crates refers the modified_at, then it will cause compile errors

Reply:
Got it